draft-ietf-quic-transport.md

```diff
@@ -3261,8 +3422,7 @@ The two type bits from a Stream ID therefore identify streams as summarized in
 | 0x3      | Server-Initiated, Unidirectional |
 {: #stream-id-types title="Stream ID Types"}
 
-Stream ID 0 (0x0) is a client-initiated, bidirectional stream that is used for
-the cryptographic handshake.  Stream 0 MUST NOT be used for application data.
+The first bi-directional stream opened by the client is stream 0.
```

Comment:
Doesn't the HTTP spec specify the stream ID for HTTP control path stuff? Might be worth double checking they don't have a specific number that needs to be updated as well.

Reply:
Not that, because the control stream is actually two unidirectional streams.  However, it does need to remove the text about QUIC reserving stream 0 for crypto and requests starting on stream 4.

Reply:
Ugh, yeah. I'm not going to have a chance to fix this.